README.md

```diff
@@ -14,7 +14,7 @@ development of software applications and accelerators using OPAE.
 General documentation on OPAE can be found at <http://01.org/OPAE>.
```

Comment:
http://01.org/OPAE change to https://opae.github.io/

Reply:
These changes are not related to the subject matter of this pull request and would be better categorized under a separate pull request.